### DISCUSSION TITLE
How to skip the particular folder itself

### DISCUSSION THREAD

**Discussion by shreesh52** [Ericsson/codechecker#4149] [General]
I tried CodeChecker analyze compile_cmds.json -o ./reports --skip skipfile.txt
and I mentioned - /path/to/projectfile/ But this is not working anyone has solution please reply....

**Comment by bruntib** [comment]
Could you attach skipfile.txt, please?

**Reply by shreesh52** [reply]
- /**/**/**/include/**/* only this line is there   ** is path not to be disclosed

**Reply by shreesh52** [reply]
the attached one is skipfile
[skipfile.txt](https://github.com/Ericsson/codechecker/files/13888336/skipfile.txt)

**Reply by bruntib** [reply]
Is it possible that the `./reports` folder already contains some reports from earlier analysis actions when the skipfile wasn't used? Could you remove this `./reports` folder and retry analysis from scratch?

If the reports in the skipped folders still appear, then could you, please, share some logs which shows that after using this skipfile there are still reports in this folder? For example the output of this command:

`CodeChecker check -l compile_cmds.json --skip skipfile.txt`

Thank you!
